### PR TITLE
chore: alb feature and enhancement

### PR DESCRIPTION
**Pull request by bemillenium**
# Submit a pull request :rocket:

Thank you for helping us contribute! Please give us more information about this PR.

---

### Adds

-variable `additional_security_group_alb_ingress_rules`

### Changes

- alb with name more than 32 will be strip

## Why :pleading_face:

- Some resource name is more than 32 cause alb cannot be created.
- To add more additional security group to alb

### CODE REVIEW

**Review by xshot9011 — COMMENTED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+### Added
+
+- variable additional_security_group_alb_ingress_rules
+
+### Changes
+
+- alb with name more than 32 will be strip
+
+
```
Comment (xshot9011) [inline CHANGELOG.md:13]: remove additional line

File: variables.tf [hunk 2]
```diff
@@ -86,6 +86,11 @@ variable "alb_aws_security_group_id" {
   description = "(Require) when is_create_alb_security_group is set to `false`"
   default     = ""
 }
```
Comment (xshot9011) [inline variables.tf:88]: Add aditional line

File: locals.tf [hunk 3]
```diff
@@ -2,7 +2,8 @@
 /*                                  Generics                                  */
 /* -------------------------------------------------------------------------- */
 locals {
-  cluster_name = "${var.prefix}-${var.environment}-${var.name}"
+  cluster_name_tmp = "${var.prefix}-${var.environment}-${var.name}"
+  cluster_name = substr("${local.cluster_name_tmp}", 0, min(19, length(local.cluster_name_tmp)))
```
Comment (xshot9011) [inline locals.tf:6]: I'm not sure this use overall resource -> only alb (I'm nor sure) need strip right, 
other resource seem ok

Comment (bemillenium) [reply locals.tf:6]: yes, I test with the long name and only alb face this problem at the moment. (limit to 32). We must to strip to 19 to concat another string.

**Review by xshot9011 — APPROVED** [review]
LGTM

**Pull request merged**